README.md

```diff
@@ -24,11 +24,11 @@ Please read this [blog post](https://tvc-16.science/hacktoberfest-lrr-2.html) fo
  
 |Main Page, Thumbnail View | Main Page, List View |
 |---|---|
-| [![archive_thumb](./tools/_screenshots/archive_thumb.jpg)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/archive_thumb.jpg) | [![archive_list](./tools/_screenshots/archive_list.png)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/archive_list.png) |
+| [![archive_thumb](./tools/_screenshots/archive_thumb.png)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/archive_thumb.jpg) | [![archive_list](./tools/_screenshots/archive_list.png)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/archive_list.png) |
 
 |Archive Reader | Reader with overlay |
 |---|---|
-| [![reader](./tools/_screenshots/reader.jpg)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/reader.jpg) | [![reader_overlay](./tools/_screenshots/reader_overlay.jpg)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/reader_overlay.jpg) |
+| [![reader](./tools/_screenshots/reader.png)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/reader.jpg) | [![reader_overlay](./tools/_screenshots/reader_overlay.png)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/reader_overlay.jpg) |
```

Comment:
```suggestion
| [![reader](./tools/_screenshots/reader.png)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/reader.png) | [![reader_overlay](./tools/_screenshots/reader_overlay.png)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/reader_overlay.png) |
```
If you change the type of the files, you'd also need to change the type in the matching link.  

Could I also convince you to convert the reader/reader overlay screenshots back to .jpgs? .pngs are way too heavy when it comes to full-size images.

Reply:
Done! They should be fine now!